core/sql.go

```diff
@@ -0,0 +1,52 @@
+package core
```

Comment:
Let's call our query language as `DSQL` instead of `SQL` because people might have different expectations when they hear `SQL`. So, change the file name to `dsql.go`.

Reply:
Done.